src/quiz/pages/Description.jsx

```diff
@@ -74,17 +64,12 @@ export default function Description() {
     }, 5000);
   }
 
+  if (!data) return <div>読み込み中</div>;
   return (
```

Comment:
```suggestion
  if (!data) return <div>読み込み中</div>;
  else {
  return (
```

がいいかも
優先されるのはわかるけど不具合も起きそう